gluten-core/src/main/scala/io/glutenproject/execution/BasicPhysicalOperatorTransformer.scala

```diff
@@ -346,8 +346,7 @@ case class ProjectExecTransformer(projectList: Seq[NamedExpression],
     }
     assert(currRel != null, "Project Rel should be valid")
 
-    val outputAttrs = BindReferencesUtil.bindReferencesWithNullable(output, inputAttributes)
```

Comment:
I find `bindReferencesWithNullable` is also used in elsewhere,  can we remove all of them?